src/care/forall.h

```diff
@@ -120,6 +120,20 @@ namespace care {
       forall(RAJA::seq_exec{}, fileName, lineNumber, start, end, std::forward<LB>(body));
    }
 
+   ////////////////////////////////////////////////////////////////////////////////
+   template <typename LB>
```

Comment:
The stream changes should be in a different branch.

Reply:
To undo all the changes in a file you can do "git checkout develop -- <filename>".

For example, "git checkout develop -- src/care/forall.h"